cmd/honeydipper/integration_test.go

```diff
@@ -109,7 +114,26 @@ func intTestProcesses(t *testing.T) {
 	pidstr, err = exec.CommandContext(ctx, "/usr/bin/pgrep", "-P", ppid).Output()
 	assert.Nil(t, err, "should be able to run pgrep to find all child processes")
 	pids := strings.Split(string(pidstr), "\n")
-	assert.Lenf(t, pids, 11, "expecting 10 child processes for honeydipper process")
+	assert.Lenf(t, pids, 18, "expecting 17 child processes for honeydipper process")
+}
+
+func intTestMakingAPICall(t *testing.T) {
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", "http://localhost:9000/api/events", nil)
+	assert.NoErrorf(t, err, "client error creating http request")
```

Comment:
Maybe a less specific test would actually be safer here? Does this test that there is no error at all, or just not this exact error message?

Reply:
right, shouldn't have any error here.